src/documentation/docs/components/Sections/AccordionSection.mdx

```diff
@@ -0,0 +1,26 @@
+import Meta from "@/components/internal/Documentation/Meta";
+import ExampleLoader from "@/documentation/utils/ExampleLoader";
+import AccordionSection from "@/components/Sections/AccordionSection";
+import PropsTable from "@/components/internal/Documentation/PropsTable";
+import PropsTableRow from "@/components/internal/Documentation/PropsTable/components/PropsTableRow";
+
+<Meta
+  title="AccordionSection"
+  subtitle="The Accordion Section consists of subsections which open and close when clicked on."
+/>
+<PropsTable>
+  <PropsTableRow value="dark" optional>
+    Specifies whether the accordion should be dark or light. Possible values are
+    <span class="code">true</span>
+    <span class="code"><strong>false</strong></span>
```

Comment:
Please add an explanation that false is the default value.

e.g. `Default is <code>false</code>`